packages/serverless-contracts/package.json

```diff
@@ -41,8 +41,10 @@
     "watch": "pnpm clean && concurrently 'pnpm:package-* --watch'"
   },
   "dependencies": {
+    "@aws-sdk/client-eventbridge": "^3.188.0",
```

Comment:
is it swarmion's responsability to have it's own version of the sdk ?
wouldn't a `peerDependency` be more appropriate here ?

Reply:
Rather a `devDependency` here, as we only use it for typing